packages/ionic/src/commands/cordova/run.ts

```diff
@@ -93,6 +94,8 @@ export class RunCommand extends CordovaCommand implements CommandPreRun {
         groups: [OptionGroup.Advanced, OptionGroup.Cordova],
         hint: 'cordova',
       },
+      // Angular build options
+      ...ANGULAR_BUILD_COMMAND_OPTIONS,
```

Comment:
These additional options should be returned in the `getCommandMetadata` method of `ServeRunner` for the angular project type so they don't show up for non-angular projects.